api/v1alpha2/types.go

```diff
@@ -181,9 +181,12 @@ var (
 
 // IngressRule defines an Azure ingress rule for security groups.
 type IngressRule struct {
-	Description string                `json:"description"`
+	Name        *string               `json:"name"`
+	Description *string               `json:"description"`
 	Protocol    SecurityGroupProtocol `json:"protocol"`
 
+	Priority *int32 `json:"priority,omitempty"`
```

Comment:
let's keep this change scoped to v1alpha3, I don't think we need to backport the fix to v1alpha2